src/config/index.ts

```diff
@@ -228,6 +230,12 @@ export const GOCD_PAUSED_PIPELINE_REMINDERS: GoCDPausedPipelineReminder[] = [
 export const SENTRY_OPTIONS_WEBHOOK_SECRET =
   process.env.SENTRY_OPTIONS_WEBHOOK_SECRET;
 export const GOCD_WEBHOOK_SECRET = process.env.GOCD_WEBHOOK_SECRET;
+export const KAFKA_CONTROL_PLANE_WEBHOOK_SECRET =
+  process.env.KAFKA_CONTROL_PLANE_WEBHOOK_SECRET;
+
+if (KAFKA_CONTROL_PLANE_WEBHOOK_SECRET === undefined) {
```

Comment:
Instead of bricking eng-pipes if this secret isn't set, could we just send respond with 500 on the specific endpoint?